src/amo/components/Home/styles.scss

```diff
@@ -105,7 +119,7 @@
 }
 
 .Home-SubjectShelf-list-item .Icon {
-  background-position: center;
+  background-position: left;
```

Comment:
hi @hritvi, something I just noticed is if it's rtl, we should have:

`background-position: right`

![Alt text](https://monosnap.com/image/J2ws3g58sT4j7nq3I9wcFo7JOVKpDU.png)

and then we just need to override this for the larger breakpoint..